Neuron/Sections/Transaction/Send/TransactonSender.swift

```diff
@@ -117,7 +85,56 @@ extension TransactonSender {
                 quota: paramBuilder.gasLimit,
                 chainId: BigUInt(paramBuilder.chainId)!,
                 password: password)
+            recordTxDetails(txhash: result.0, blockNumber: result.1)
             return result.0
         }
     }
+
+    func recordTxDetails(txhash: String, blockNumber: BigUInt? = nil) {
+        let cetateModelBlock = { () -> LocalTxDetailModel in
+            switch self.paramBuilder.tokenType {
+            case .ether:
+                return LocalTxDetailModel(
+                    tokenIdentifier: self.paramBuilder.tokenIdentifier,
+                    txHash: txhash,
+                    from: self.paramBuilder.from, to: self.paramBuilder.to,
+                    value: self.paramBuilder.value,
+                    gasPrice: self.paramBuilder.gasPrice,
+                    gasLimit: BigUInt(self.paramBuilder.gasLimit)
```

Comment:
@cezres It seems the right time do change `paramBuilder.gasLimit` type to `BigUInt`, how do you think? If that's reasonable could you change that (with another PR)?